### PR TITLE
Change quotes to apostrophes

### PR DESCRIPTION
**Pull request by viogustian**
i have completed my assignment, thank you please approve

### CODE REVIEW

**Review by curquiza — CHANGES_REQUESTED** [review]
This change should only applied to literal strings.
Strings where there are interpretation should keep the `"` quote.
Ex: strings with `\em` and string with `$`.

**Comment by viogustian** [comment]
Ok, i will fix it

**Comment by viogustian** [comment]
I've fixed it

File: scripts/deploy-meilisearch.sh [hunk 1]
```diff
@@ -4,16 +4,16 @@ mkdir /var/opt/meilisearch/dumps
 git clone https://github.com/meilisearch/cloud-scripts.git /tmp/meili-tmp
 cd /tmp/meili-tmp # DELETE?
 
-git checkout "$1"
+git checkout '$1'
```
Comment (alallema) [inline scripts/deploy-meilisearch.sh:7]:
```suggestion
git checkout "$1"
```

File: scripts/deploy-meilisearch.sh [hunk 2]
```diff
@@ -4,16 +4,16 @@ mkdir /var/opt/meilisearch/dumps
...
 chmod 755 /tmp/meili-tmp/scripts/first-login/*
 chmod 755 /tmp/meili-tmp/scripts/MOTD/*
 cp -r /tmp/meili-tmp/scripts/first-login/* /var/opt/meilisearch/scripts/first-login/.
-sed -i "s/provider_name/$2/" /var/opt/meilisearch/scripts/first-login/000-set-meili-env.sh
+sed -i 's/provider_name/$2/' /var/opt/meilisearch/scripts/first-login/000-set-meili-env.sh
```
Comment (alallema) [inline scripts/deploy-meilisearch.sh:11]:
```suggestion
sed -i "s/provider_name/$2/" /var/opt/meilisearch/scripts/first-login/000-set-meili-env.sh
```

File: scripts/first-login/000-set-meili-env.sh [hunk 3]
```diff
@@ -51,7 +51,7 @@ ask_production_environment() {
         case $yn in
             [Yy]* ) set_production_env=true; set_master_key=true; break;;
             [Nn]* ) set_production_env=false; break;;
-            * ) echo "  Please answer by writting 'y' for yes or 'n' for no.";
+            * ) echo '  Please answer by writting 'y' for yes or 'n' for no.';
```
Comment (alallema) [inline scripts/first-login/000-set-meili-env.sh:54]:
```suggestion
            * ) echo "  Please answer by writing 'y' for yes or 'n' for no.";
```

File: scripts/first-login/000-set-meili-env.sh [hunk 4]
```diff
@@ -62,20 +62,20 @@ ask_master_key_setup() {
         case $yn in
             [Yy]* ) set_master_key=true; break;;
             [Nn]* ) set_master_key=false; break;;
-            * ) echo "  Please answer by writting 'y' for yes or 'n' for no.";
+            * ) echo '  Please answer by writting 'y' for yes or 'n' for no.';
```
Comment (alallema) [inline scripts/first-login/000-set-meili-env.sh:65]:
```suggestion
            * ) echo "  Please answer by writing 'y' for yes or 'n' for no.";
```

File: scripts/first-login/000-set-meili-env.sh [hunk 5]
```diff
@@ -62,20 +62,20 @@ ask_master_key_setup() {
...
     if [ "$MEILISEARCH_MASTER_KEY" != "" ]; then
-        echo $BOLD$GREEN"A previous MEILI_MASTER_KEY has been detected. It was set to" $MEILISEARCH_MASTER_KEY$RESET
+        echo $BOLD$GREEN'A previous MEILI_MASTER_KEY has been detected. It was set to' $MEILISEARCH_MASTER_KEY$RESET
         while true; do
         read -p "$(echo $BOLD$BLUE"Do you wish to keep using this MEILI_MASTER_KEY for your search engine [y/n]?  "$RESET)" yn
         case $yn in
             [Yy]* ) keep_previous_master_key=true; break;;
             [Nn]* ) keep_previous_master_key=false; break;;
-            * ) echo "  Please answer by writting 'y' for yes or 'n' for no.";
+            * ) echo '  Please answer by writting 'y' for yes or 'n' for no.';
```
Comment (alallema) [inline scripts/first-login/000-set-meili-env.sh:78]:
```suggestion
            * ) echo "  Please answer by writing 'y' for yes or 'n' for no.";
```

File: scripts/first-login/000-set-meili-env.sh [hunk 6]
```diff
@@ -87,9 +87,9 @@ generate_master_key() {
     while true; do
         read -p "$(echo $BOLD$BLUE"Do you wish to specify your MEILI_MASTER_KEY (otherwise it will be generated) [y/n]? "$RESET)" yn
         case $yn in
-            [Yy]* ) read -p "MEILI_MASTER_KEY: " api_key; break;;
-            [Nn]* ) api_key=$(date +%s | sha256sum | base64 | head -c 32); echo "You MEILI_MASTER_KEY is $api_key"; echo "You should keep it somewhere safe."; break;;
-            * ) echo "  Please answer by writting 'y' for yes or 'n' for no.";;
+            [Yy]* ) read -p 'MEILI_MASTER_KEY: ' api_key; break;;
+            [Nn]* ) api_key=$(date +%s | sha256sum | base64 | head -c 32); echo "You MEILI_MASTER_KEY is $api_key"; echo 'You should keep it somewhere safe.'; break;;
+            * ) echo '  Please answer by writting 'y' for yes or 'n' for no.';;
```
Comment (alallema) [inline scripts/first-login/000-set-meili-env.sh:92]:
```suggestion
            * ) echo "  Please answer by writing 'y' for yes or 'n' for no.";;
```

File: scripts/first-login/000-set-meili-env.sh [hunk 7]
```diff
@@ -100,7 +100,7 @@ ask_domain_name_setup() {
         case $yn in
             [Yy]* ) ask_domain_name=true; break;;
             [Nn]* ) ask_domain_name=false; break;;
-            * ) echo "  Please answer by writting 'y' for yes or 'n' for no.";;
+            * ) echo '  Please answer by writting 'y' for yes or 'n' for no.';;
```
Comment (alallema) [inline scripts/first-login/000-set-meili-env.sh:103]:
```suggestion
            * ) echo "  Please answer by writing 'y' for yes or 'n' for no.";;
```

File: scripts/first-login/000-set-meili-env.sh [hunk 8]
```diff
@@ -121,7 +121,7 @@ ask_ssl_configure() {
         case $yn in
             [Yy]* ) want_ssl_certbot=true; break;;
             [Nn]* ) want_ssl_certbot=false; break;;
-            * ) echo "  Please answer by writting 'y' for yes or 'n' for no.";
+            * ) echo '  Please answer by writting 'y' for yes or 'n' for no.';
```
Comment (alallema) [inline scripts/first-login/000-set-meili-env.sh:124]:
```suggestion
            * ) echo "  Please answer by writing 'y' for yes or 'n' for no.";
```

File: scripts/first-login/000-set-meili-env.sh [hunk 9]
```diff
@@ -132,14 +132,14 @@ ask_has_own_ssl() {
         case $yn in
             [Yy]* ) has_own_ssl=true; break;;
             [Nn]* ) has_own_ssl=false; break;;
-            * ) echo "  Please answer by writting 'y' for yes or 'n' for no.";
+            * ) echo '  Please answer by writting 'y' for yes or 'n' for no.';
```
Comment (alallema) [inline scripts/first-login/000-set-meili-env.sh:135]:
```suggestion
            * ) echo "  Please answer by writing 'y' for yes or 'n' for no.";
```

File: scripts/first-login/000-set-meili-env.sh [hunk 10]
```diff
@@ -174,8 +174,8 @@ ask_domain_name_setup
 
 if [ $ask_domain_name != true ]; then
     DOMAIN_NAME=""
```
Comment (alallema) [inline scripts/first-login/000-set-meili-env.sh:176]:
```suggestion
    DOMAIN_NAME=''
```

File: scripts/first-login/001-setup-prod.sh [hunk 11]
```diff
@@ -23,7 +23,7 @@ exit_with_message() {
     echo ""
     echo "$BOLD If you want to run this script again, run the following command:$RESET"
     echo ""
-    echo " meilisearch-setup"
+    echo "meilisearch-setup"
```
Comment (alallema) [inline scripts/first-login/001-setup-prod.sh:26]:
```suggestion
    echo " meilisearch-setup"
```

**Review by alallema — COMMENTED** [review]
Thanks for this PR @viogustian. I made some suggestions

**Comment by viogustian** [comment]
thanks

**Comment by curquiza** [comment]
Thanks for your changes @viogustian!
I let @alallema do the final review :)

**Comment by viogustian** [comment]
when will my pull request be merged?

**Comment by curquiza** [comment]
When she has the time to review, be patient, you can understand we have a lot of work 🙂

**Review by alallema — APPROVED** [review]
LGTM!

**Pull request merged**